tests/client-units/account_store.coffee

```diff
@@ -44,8 +44,8 @@ describe 'AccountStore initialized without account', ->
     #     RouterStore.isWaiting().should.be.false
 
     # it 'Then AccountStore should have a nomailboxes error', ->
-    #     noMailboxErr = RouterStore.getErrors().get('nomailboxes')
-    #     noMailboxErr.message.should.equal 'translated config error nomailboxes'
+    #    noMailboxErr = RouterStore.getErrors().get('nomailboxes')
```

Comment:
What about removing them? They are here for long. I'm not sure it makes sense to keep commmented tests.